python/tskit/vcf.py

```diff
@@ -50,17 +50,21 @@ class VcfWriter(object):
     file-like object.
     """
     def __init__(
-            self, tree_sequence, ploidy=None, contig_id="1", individual_names=None,
-            position_transform=None):
+            self, tree_sequence, ploidy=None, contig_id="1", individuals=None,
+            individual_names=None, position_transform=None):
         self.tree_sequence = tree_sequence
         self.contig_id = contig_id
 
+        if individuals is None:
+            individuals = list(range(self.tree_sequence.num_individuals))
```

Comment:
``np.arange(tree_sequence.num_individuals, dtype=int)`` would be a bit more efficient here.

Reply:
Still not sure whether to always numpy or only when necessary...

Reply:
The rule I'm using at the moment is, use numpy *except* if the numpy version is hard to read and the perf implications aren't serious (the reason I didn't use your numpy version of the positions transform is that it didn't have exactly the same semantics, and it didn't seem that important, given limited time). Here, arange is exactly as readable as list(range()), so no reason not to use numpy.